src/pages/about/nft-book.vue

```diff
@@ -0,0 +1,978 @@
+          fill="#7FD6D2"
+        />
+      </svg>
+    </section>
+
+    <section id="store" class="px-[2rem] pt-[3rem] pb-[6rem]">
+      <div class="w-full mx-auto max-w-[1024px] space-y-[2rem]">
+        <h2 class="text-[1.5rem] text-center font-irohamaru tracking-[0.5rem]">
+          {{ $t('about_nft_book_section_store_title') }}
+        </h2>
+
+        <NFTBookShelf
+          :items="nftBooks"
+          theme="future"
+          @click-item="onClickShelfItem"
+          @click-item-avatar="onClickShelfItemAvatar"
+        />
+
+        <footer class="flex justify-center">
+          <ButtonV2
+            :text="$t('about_nft_book_section_store_view_more')"
+            :to="localeLocation({ name: 'store' })"
+            theme="glow"
+            preset="secondary"
+            @click.native="handleCTAClickInNFTBookSection"
+          />
+        </footer>
+      </div>
+    </section>
+  </main>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+import { logTrackerEvent } from '~/util/EventLogger';
+import { LIKECOIN_NFT_BOOK_INDEX_FEATURED_ITEMS } from '~/constant';
+
+import WaveBg from '~/assets/images/about/nft-book/wave.svg';
+
+export default {
+  name: 'AboutNFTBookPage',
+  async fetch({ store }) {
+    try {
+      await store.dispatch('fetchBookstoreList');
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
+    }
+  },
+  computed: {
+    ...mapGetters(['nftGetBookstoreListItems']),
+    bannerOverlayBG() {
+      return WaveBg;
+    },
+    bookSectionClassId() {
+      return LIKECOIN_NFT_BOOK_INDEX_FEATURED_ITEMS[0];
+    },
+    nftBooks() {
+      return [
+        ...this.nftBooksDisplayInFullWidth,
+        ...this.nftBooksDisplayOnShelf,
+      ].slice(0, 4);
+    },
+    nftBooksDisplayInFullWidth() {
+      return this.nftGetBookstoreListItems('highlighted').filter(nft =>
+        // Display books of the current locale only
+        this.$i18n.locale.includes(nft.locale)
+      );
+    },
+    nftBooksDisplayOnShelf() {
+      const { locale } = this.$i18n;
+      const books = [...this.nftGetBookstoreListItems('featured')];
+      // Display books of the current locale first
+      books.sort((a, b) => {
+        const aMatchedLocale = locale.includes(a.locale);
+        const bMatchedLocale = locale.includes(b.locale);
+        if (aMatchedLocale && !bMatchedLocale) {
+          return -1;
+        }
+        if (!aMatchedLocale && bMatchedLocale) {
+          return 1;
+        }
+        return 0;
```

Comment:
Did we change this part of logic in another part of code in another PR?

Reply:
Yes, this should be merged first